packages/babel-plugin/src/__tests__/module-traversal.test.tsx

```diff
@@ -134,7 +131,9 @@ describe('module traversal', () => {
     `
     );
 
-    expect(result).toInclude('.cc-hash-test{font-size:14px;color:blue;background:red}');
+    expect(result).toInclude('{font-size:14px}');
+    expect(result).toInclude('{color:blue}');
+    expect(result).toInclude('{background:red}');
```

Comment:
nit: `toInclude` can be replaced with `toIncludeMultiple` for multiple substrings.
https://github.com/jest-community/jest-extended#toincludemultiplesubstring

Reply:
nice will do tmr

Reply:
i like it but it takes a while to make the change - for now im just going to leave it.